cfgov/paying_for_college/handlers.py

```diff
@@ -0,0 +1,6 @@
+from corsheaders.signals import check_request_enabled
+
+def cors_allow_particular_urls(sender, request, **kwargs):
```

Comment:
One terribly minor thing, could we include a comment above this handler about why we're allowing any origin here (to explicitly permit the use of this API from outside), and maybe a note that that's what this is doing, allowing all origins (the `check_request_enabled` signal name isn't exactly descriptive of that, and this is the first time we're using a pattern like this).